tests/integration/api_v2/subscribeApi/feesEvent.test.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const config = require('../../../config');
+
+const {
+	subscribeAndReturn,
+	closeAllConnections,
+} = require('../../../helpers/socketIoSubscribe');
+
+const {
+	feeEstimateEventSchema,
+} = require('../../../schemas/api_v2/fees.schema');
+
+const endpoint = `${config.SERVICE_ENDPOINT_RPC}/blockchain`;
+
+describe('Test subscribe API fees estimates event', () => {
+	it('event update.fee_estimates', async () => {
+		const response = await subscribeAndReturn(endpoint, 'update.fee_estimates');
+		expect(response).toMap(feeEstimateEventSchema);
```

Comment:
@MichalTuleja Unlike the remaining events, the data payload associated with the `update.fee_estimates` event seems to be not enclosed within the standard response envelope and is inconsistent. Should we standardize it?

Reply:
Yes, please make it so:

```
{ "data": {
  "blockHeight": 12463,
  "low": 0,
  "med": 0,
  "high": 0,
  "updated": 1622816707,
  "blockId": "940c71fed31d45e5087ad63d494224d3417e32f44ddef6c182dfd0258994e3ea",
  "minFeePerByte": 1000,
  "baseFeeByModuleAssetId": {
    "5:0": "1000000000"
  },
  "baseFeeByModuleAssetName": {
    "dpos:registerDelegate": "1000000000"
  }
}}
```

Reply:
Another question: why is `baseFeeByModuleAssetId` only showing "5:0"? Are other transactions returned without the base fee?

Reply:
@MichalTuleja Both `baseFeeByModuleAssetId` and `baseFeeByModuleAssetName` are generated dynamically based on the information returned by the node (`apiClient.node.getNodeInfo()`) to which the core service connects to.